apps/web/src/contexts/globalLayout/GlobalAnnouncementPopover/useGlobalAnnouncementPopover.tsx

```diff
@@ -55,7 +55,7 @@ export default function useGlobalAnnouncementPopover({
   // NOTE: next time we use global announcements, we'll need to set a new flag in the schema
   // const isGlobalAnnouncementExperienced = true;
   const [isGlobalAnnouncementExperienced, setGlobalAnnouncementExperienced] = useExperience({
-    type: 'PostsBetaAnnouncement',
+    type: 'CreatorBetaFullscreenAnnouncementModal',
```

Comment:
Seems like the `typecheck` failed because of missing this key. Maybe we need to update the relay schema

Reply:
good catch, updated!